cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -2205,122 +2298,117 @@ features:
+        description: Upload your application JAR file
         filePathLabel: Choose file
-        shorttitle: Upload Application
-        title: Upload JAR
-        uploadHelperText: Upload the JAR for the application that you wish to deploy
+        shorttitle: Upload application
+        title: Upload JAR file
+        uploadHelperText: Upload the JAR file for the application that you wish to deploy
       success: You have successfully deployed the application "{appName}".
     ArtifactUpload:
       callToAction: Create a Pipeline
-      headerlabel: Add Third Party Driver
-      footertitle: Add Driver
+      headerlabel: Add third party driver
+      footertitle: Add driver
       Step1:
-        description: Upload your Driver
-        shorttitle: Upload Driver
-        title: Upload JAR
-        uploadHelperText: Upload the third party driver that was downloaded in the previous step
+        description: Upload your driver
         filePathLabel: Choose file
+        shorttitle: Upload driver
+        title: Upload JAR  file
+        uploadHelperText: Upload the third party driver that was downloaded in the previous step
       Step2:
-        classnameLabel: Class Name
-        classnamePlaceholder: Driver Class Name. E.g. com.example.MyClass
+        classnameLabel: Class name
+        classnamePlaceholder: Driver class name. E.g. com.example.MyClass
         description: Configure the settings for your driver
         descriptionLabel: Description
-        decriptionPlaceholder: Driver Description
+        descriptionPlaceholder: Driver description
         nameLabel: Name
-        namePlaceholder: Driver Name
-        parentArtifactLabel: Parent Artifact
-        shorttitle: Driver Configuration
-        title: Configure Driver
+        namePlaceholder: Driver name
+        parentArtifactLabel: Parent artifact
+        shorttitle: Driver configuration
+        title: Configure driver
       success: You have successfully uploaded the driver "{artifactName}".
-      subtitle: You can now create a pipeline to extract data from Database using the driver.
+      subtitle: You can now create a pipeline to extract data from database using the driver.
     DirectiveUpload:
-      callToAction: Go to Preparation
-      footertitle: Upload Directive Artifact
+      callToAction: Go to preparation
+      footertitle: Upload directive artifact
       Step1:
-        description: Upload your Directive JAR.
+        description: Upload your directive JAR file.
         errorMessage: Invalid directive. Directive must be a JAR file.
-        filePathLabel: Choose File
-        shorttitle: Upload Directive JAR
-        title: Upload Directive JAR
+        filePathLabel: Choose file
+        shorttitle: Upload directive JAR file
+        title: Upload directive JAR file
       Step2:
-        description: Upload the directive configuration JSON.
+        description: Upload the directive configuration JSON file.
```

Comment:
do we have a standard about using periods at the end?

Reply:
Asked Lea about this, we decided to make it consistent by adding periods at the end for all wizard step descriptions.